### PR TITLE
dnn: remove raw SSE2 implementation

### PR DESCRIPTION
**Pull request by tomoaki0705**
### This pullrequest changes
 - remove raw SSE2 implementation
 - Unfortunately, neither test_dnn nor perf_dnn covers this part
 - Adding new new test case for these part seems really complicated task, so I'll just throw the PR

### CODE REVIEW

**Comment by vpisarev** [comment]
@tomoaki0705, thanks!

File: modules/dnn/src/layers/pooling_layer.cpp [hunk 1]
```diff
@@ -488,19 +488,18 @@ class PoolingLayerImpl CV_FINAL : public PoolingLayer
...
-#if CV_SSE2
                                     else if( stride_w == 2 )
                                         for (int k = 0; k < kernel_w*kernel_h; k++)
                                         {
                                             int index = ofsptr[k];
-                                            v_float32x4 v00 = v_load(srcData1 + index), v01 = v_load(srcData1 + index + 4);
-                                            v_float32x4 v0(_mm_shuffle_ps(v00.val, v01.val, _MM_SHUFFLE(2, 0, 2, 0)));
-                                            v_float32x4 v10 = v_load(srcData1 + index + 8), v11 = v_load(srcData1 + index + 12);
-                                            v_float32x4 v1(_mm_shuffle_ps(v10.val, v11.val, _MM_SHUFFLE(2, 0, 2, 0)));
+                                            v_int32x4 vs320, vs321, dummy;
```
Comment (vpisarev) [inline modules/dnn/src/layers/pooling_layer.cpp]: why not use `v_float32x4`?

Comment (tomoaki0705) [reply modules/dnn/src/layers/pooling_layer.cpp]: Nice catch !
I just found ```v_int32x4``` and didn't really look for ```v_float32x4```
Thank you !

**Comment by vpisarev** [comment]
thanks! :+1:

**Pull request merged**